docs/QbeastTable.md

```diff
@@ -26,13 +26,19 @@ qbeatsTable.lastRevisionID() // the last Revision identifier
 ```
 
 ## Table Operations
+
+### Optimization
 Through `QbeastTable` you can also execute the `Optimize` operation. This command is used to **rearrange the files in the table** according to the dictates of the index to render queries more efficient. 
 
-These are a few different ways of executing the `optimize` operation, with the input parameters being:
-1. `revisionID`: The revision number you want to optimize, the default is the latest revision.
-2. `fraction`: The fraction of the data of the specified revision you want to optimize.
-3. `options`: A map of options of the optimization. You can specify the `userMetadata` as well as configurations for `io.qbeast.spark.delta.hook.PreCommitHook.`
+#### Paramaters of Manual Optimization
+| Parameter    | Description                                                                                               | Default            |
+|--------------|-----------------------------------------------------------------------------------------------------------|--------------------|
+| `revisionID` | The revision number you want to optimize.                                                                 | Latest revision    |
+| `fraction`   | The fraction of the data of the specified revision you want to optimize.                                  | None specified     |
+| `options`    | A map of options for optimization. You can specify `userMetadata` and configurations for `PreCommitHook`. | None specified     |
+
 
+#### Examples of Manual Optimization
 ```scala
 // Optimizing 10% of the data from Revision number 2, and stores some user metadata
 qbeastTable.optmize(2L, 0.1, Map["userMetadata" -> "user-metadata-for-optimization"])
```

Comment:
`qbeastTable.optmize(2L, 0.1, Map("userMetadata" -> "user-metadata-for-optimization"))`
I think it was a mistake I made in a previous PR. Use `Map()` instead of `Map[]`

Reply:
done in 8cab2b8